amf-client/shared/src/test/scala/amf/client/validation/ClientPayloadValidationTest.scala

```diff
@@ -227,5 +227,14 @@ trait ClientPayloadValidationTest extends AsyncFunSuite with NativeOps with Matc
     }
   }
 
+  test("Long type with int64 format is validated as long") {
+    amf.Core.init().asFuture.flatMap { _ =>
+      amf.Core.registerPlugin(PayloadValidatorPlugin)
+      val shape = new ScalarShape().withDataType(DataTypes.Long).withFormat("int64")
```

Comment:
Will this change have the same impact for other formats such as int32, int, long, int16, int8?

Reply:
This will just impact int64 and long. They are aliases

Reply:
int, int8, int16 and int32 are all parsed to xsd integer type